NEWS.md

```diff
@@ -7,6 +7,11 @@
 * Added {systemfonts} as a dependency in Suggests as it is required to build the
 vignette in **r-oldrel-macos-arm64** (#218, @Karim-Mane).
 
+## Enhancements
+
+* The value of the `duplicates_checked_from` element of the report is now a
+vector of target columns (#228, @Karim-Mane).
```

Comment:
Could be worth mentioning that it used to be a single `character` string to make clear how it being a vector is different to previous versions.